main.go

```diff
@@ -80,7 +78,9 @@ func serve(cfg *config.Configuration) error {
 	}
 
 	corsRouter := router.SupportCORS(r)
-	server.Listen(cfg, router.NoCache{Handler: corsRouter}, router.Admin(currencyConverter, fetchingInterval), r.MetricsEngine)
+	if err := server.Listen(cfg, router.NoCache{Handler: corsRouter}, router.Admin(currencyConverter, fetchingInterval), r.MetricsEngine); err != nil {
+		glog.Errorf("prebid-server returned an error: %v", err)
```

Comment:
Good catch on this error. I think this be a fatal "terminate now" error.

Reply:
changing to `glog.Fatalf("prebid-server returned an error: %v", err)`